app/views/shared/_zendesk_snippet.html.erb

```diff
@@ -1,5 +1,9 @@
-<script
-  id="ze-snippet"
-  nonce="<%= request.content_security_policy_nonce %>"
-  src=https://static.zdassets.com/ekr/snippet.js?key=904d04ae-8027-4ebb-ad39-d0e9f68d6ab1>
-</script>
+<% if Rails.env.test? %>
+  <%= javascript_pack_tag('zendesk-stub') %>
+<% else %>
+  <script
+    id="ze-snippet"
+    nonce="<%= request.content_security_policy_nonce %>"
+    src=https://static.zdassets.com/ekr/snippet.js?key=904d04ae-8027-4ebb-ad39-d0e9f68d6ab1>
```

Comment:
Is this an API token? Should we put it in a env variable?

Reply:
Good idea — will amend, and we'll see about refreshing the token now it's been in here

Reply:
Not sure around this, seeing as this will always be accessible in the page source even when in an ENV variable? Also I don't believe this is different between different ENVs and is a static key

Reply:
Good point @cwrw!

Reply:
Ah yeah, I was just wondering about this really. If it never changes then it's fine as is. If we put it in an env var then we don't have to PR again when it changes